manifests/cluster-rbac/server/argocd-server-clusterrole.yaml

```diff
@@ -32,6 +32,7 @@ rules:
   - "argoproj.io"
   resources:
   - "applications"
+  - "applicationsets"
```

Comment:
Hm. I think this is already in the Role. Shouldn't that be sufficient? The `ApplicationSets` resources should always live in the same namespace as the argocd-server.

Reply:
I updated it with the latest install.yaml, the appset list is good, but there is still a problem with the creation, I don't know where the problem is
```
$ argocd appset create appset-test.yaml
FATA[0000] rpc error: code = PermissionDenied desc = permission denied: applicationsets, get, default/cluster-addons, sub: admin, iat: 2022-09-16T15:27:05Z
$
$ argocd version
argocd: v2.5.0+6a44732
  BuildDate: 2022-09-16T13:58:59Z
  GitCommit: 6a44732966e03261e0e1607637c5fc73eb3a573c
  GitTreeState: clean
  GoVersion: go1.18.4
  Compiler: gc
  Platform: linux/amd64
argocd-server: v2.5.0+e53ab49
  BuildDate: 2022-09-14T07:42:23Z
  GitCommit: e53ab49cb5f5647973b32fa956170f8db7111340
  GitTreeState: clean
  GoVersion: go1.18.6
  Compiler: gc
  Platform: linux/amd64
  Kustomize Version: v4.5.5 2022-05-20T20:25:40Z
  Helm Version: v3.9.3+g414ff28
  Kubectl Version: v0.24.2
  Jsonnet Version: v0.18.0
```

Reply:
@crenshaw-dev Is there any documentation about the permissions of `argo appset`, or how can I use this command smoothly, thanks

Reply:
I don't think there's any specific documentation, but the general RBAC documentation should be sufficient.